WORKSPACE

```diff
@@ -29,6 +30,7 @@ load("//tensorflow:workspace.bzl", "tf_workspace")
 #    name="androidndk",
 #    path="<PATH_TO_NDK>",
 #    api_level=14)
+#Note that API level is not the same as NDK version.
```

Comment:
Yes, TF master. I made some edits to this file internally to add comments here as well, and those changes have been pushed since you made your edits.